test/rules/no-duplicate-variable/test.ts.lint

```diff
@@ -108,6 +109,7 @@ function testArguments1(arg: number, arg: number): void {
 // failure: local var/let declarations shadow arguments.
 function testArguments2(x: number, y: number): void {
     var x = 1;
+        ~ [err % ('x')]
     let y = 2;
```

Comment:
this rule doesn't handle block scoped declarations. these result in a compiler error, so there is no need for a linter error